editor/src/frontend/frontend_message_handler.rs

```diff
@@ -4,16 +4,23 @@ use crate::tool::tool_options::ToolOptions;
 use crate::Color;
 use serde::{Deserialize, Serialize};
 
+#[derive(PartialEq, Clone, Deserialize, Serialize, Debug)]
+pub struct FrontendDocumentState {
```

Comment:
What would you think about renaming this from -`State` to -`Status` or -`Details`, or perhaps you can think of another better name? I'm a little worried "state" implies statefulness as it pertains to the application architecture, but really this is more of just a "status" or its current "details" not related to statefulness.